README.md

```diff
@@ -88,7 +88,7 @@ The documentation is divided into several sections:
 
 ## Rename from "Hermione"
 
-This project was formerly known as "Hermione", but eventually some copyright and trademark issues surfaced, leading to the decision to rebrand. After some discussion, we settled on "Testplane" as the official new title. We've tailored Testplane `v1` to be a drop-in replacement for Hermione `v8`.
+This project was formerly known as "Hermione", but eventually some copyright and trademark issues surfaced, leading to the decision to rebrand. After some discussion, we settled on "Testplane" as the official new title. And since this is only a rebranding, we decided to implement continuous versioning instead of starting the versioning from the beginning. For this reason, Testplane `v8.x` is a drop-in replacement for Hermione `v8.x`.
```

Comment:
I'd word it along the lines of `Considering this change as merely a rebranding, we've proceeded with the existing version count instead of starting anew. Thus, ...`, because "And since this is only..." feels a bit artificial, but feel free to keep your variant — not critical to me.

Reply:
I like `Considering...` variant too.

Reply:
already fixed, tnx